### ISSUE TITLE
Theme conf "dropdown.enabled"

### ISSUE THREAD

**Issue by air-guillaume** [getgrav/grav-theme-receptar#5]
Hi there,

Thanks for this great theme. Just one thing though. Could you integrate by default the option "dropdown.enabled" ? I can't figure where to put this parameter and make menu items having children.

By default to "false" (as it is like now, without the parameter).

Would be much appreciated, thanks :).

Best regards

**Comment by flaviocopes** [comment]
The theme does not really implement dropdown menus, but can implement having the pages visible children being shown (all the time)

E.g.

![screen shot 2016-04-04 at 11 27 36](https://cloud.githubusercontent.com/assets/11783388/14243886/52ee5bb0-fa58-11e5-8a44-a0cd05d188c0.png)

Is this what you need?

**Comment by air-guillaume** [comment]
Yup ! This would be great to have something like this! (without dropdown it's even better, just having a level / indent to the right).

**Comment by flaviocopes** [comment]
This PR https://github.com/getgrav/grav-theme-receptar/pull/6 adds a menu to the admin-side theme configuration to enable such option. Give it a try
